inputs-required.tf

```diff
@@ -7,6 +7,15 @@ variable "common_tags" {
   description = "Common tag to be applied to resources"
   type        = map(string)
 }
+variable "builtFrom" {
+  type        = string
+  description = "The name of the repository these resources are builtFrom."
+}
+
+variable "product" {
+  type        = string
+  description = "The name of the prodcut this infrastructure supports."
+}
```

Comment:
Can these be remove (with the common tags module call) and just use the `var.common_tags` variable for tags.